tests/test_fields.py

```diff
@@ -678,6 +678,17 @@ class F(Form):
             "</ul>"
         )
 
+    def test_render_kw_preserved(self):
+        F = make_form(
+            a=RadioField(
+                choices=[(True, "yes"), (False, "no")],
+                render_kw=dict(disabled=True)
+            )
+        )
+        form = F()
+        assert 'disabled' in form.a()
```

Comment:
This assertion is true even without your patch. Please write a test that fails without your patch, and succeeds with your patch.

Reply:
I wrote a test to check for exact match - I don't know how to check it otherwise.